exercises/concept/treasure-chest/src/TreasureChest.elm

```diff
@@ -0,0 +1,28 @@
+module TreasureChest exposing (..)
+
+{-| One. define a single variant generic TreasureChest custom type with associated data:
+
+  - a String (for a password)
+
+        - a generic (for the treasure)
```

Comment:
The formatting is strange :D

Reply:
Remove the "-" and rewrite as a normal sentence?
Anything is better than this, it just looks off.